.travis.yml

```diff
@@ -2,7 +2,9 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "iojs"
+  - "4"
+  - "6"
+  - "node"
```

Comment:
`node` will automatically pick up next stable release, but that might be superfluous here.